engine/fake_exchange_test.go

```diff
@@ -27,6 +27,8 @@ const (
 // the engine package
 type FakePassingExchange struct {
 	exchange.Base
+
+	Holdings account.Holdings
```

Comment:
This doesn't seem right. No other exchange implementation has its own Holdings property. For the purpose of this `fakePassingExchange`, we don't really need to actually go through any motions of processing holdings, just to return a successful result (even if empty)

Reply:
Is it fine that it becomes not possible to properly test UpdateAccountInfo function?
I mean, that to test UpdateAccountInfo the flow should be as the following:
- get account info
- update account info
- get account info 
- check it's changed

GetAccountInfo calls FetchAccountInfo which is the same for every exchange.

Reply:
removed